### PR TITLE
Fix build issues with new release

### PR DESCRIPTION
**Pull request by dkanada**

### CODE REVIEW

File: Trakt/Api/TraktApi.cs [hunk 1]
```diff
@@ -118,7 +118,7 @@ public async Task<TraktScrobbleResponse> SendMovieStatusUpdateAsync(Movie movie,
             var movieData = new TraktScrobbleMovie
             {
                 app_date = DateTimeOffset.Now.Date.ToString("yyyy-MM-dd"),
-                app_version = _appHost.ApplicationVersion.ToString(CultureInfo.InvariantCulture),
+                app_version = _appHost.ApplicationVersion.ToString(),
```
Comment (JustAMan) [inline Trakt/Api/TraktApi.cs]: Why?

File: Trakt/Trakt.csproj [hunk 2]
```diff
@@ -2,21 +2,17 @@
...
-    <None Remove="thumb.png" />
-  </ItemGroup>
-
-  <ItemGroup>
     <EmbeddedResource Include="Configuration\configPage.html" />
   </ItemGroup>
 
   <ItemGroup>
-    <PackageReference Include="Jellyfin.Controller" Version="10.5-*" />
+    <PackageReference Include="Jellyfin.Controller" Version="10.5.*" />
```
Comment (JustAMan) [inline Trakt/Trakt.csproj]: Why?

File: Trakt/Api/TraktApi.cs (hunk 1 above)
Comment (Bond-009) [reply Trakt/Api/TraktApi.cs]: ApplicationVersionString exists, use that

File: Trakt/Trakt.csproj (hunk 2 above)
Comment (dkanada) [reply Trakt/Trakt.csproj]: This is how we should be matching versions, because the dash only matches pre-releases.

**Pull request merged**